.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,14 @@
+### Milestone (Odoo version)
```

Comment:
Well, I don't know how this works.

I always put relevant commits messages (for example, the README.rst in new files), and if this overwrites current behavior that show these messages, I prefer not to have it, because I will need again to copy/paste these messages.